internal/compiler/widgets/cupertino-base/spinner.slint

```diff
@@ -0,0 +1,24 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+import { Palette } from "styling.slint";
+import { SpinnerBase } from "../common/spinner-base.slint";
+
+export component Spinner {
+    in property <float> progress <=> base.progress;
+    in property <bool> indeterminate <=> base.indeterminate;
+
+    min-width: 30px;
+    min-height: 30px;
+    horizontal-stretch: 0;
+    vertical-stretch: 0;
+    accessible-role: none;
+    accessible-value: root.progress;
```

Comment:
What is the meaning of `none` as accessible role but `value` being set to the progress?

Reply:
I see that this is copy & paste from the ProgressIndicator. Nevermind :)

Reply:
Fixed in https://github.com/slint-ui/slint/commit/a5c3692fbc14eca877b1ed2ad46fc6925eaddc69 ;-)

Reply:
thx, is now used also for the spinner